locales/en/moves.json

```diff
@@ -5,6 +5,7 @@
   "steps": {
     "person_search": {
       "heading": "Who is being moved?",
+      "heading_assign_to_allocation": "Who do you want to add to this allocation?",
```

Comment:
Could we maybe use consistent terms for these variables. When @solidgoldpig did the original work we are using the term `assign` rather than `assign_to_allocation`. This also fits better the mental model of what is _actually_ happening. 

A person is not actually being assigned to an allocation, rather to the move that was previously created _by_ the allocation. This is why we moved the logic to assign a person to the move part of the application and away from allocation.

Think of the allocation of just a shortcut way to manage a group of moves expected to happen to/from the same place and on the same day. The further actions around adding/removing a person are done on specific moves.